pkg/config/config.go

```diff
@@ -532,6 +532,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("cloud_foundry_bbs.ca_file", "")
 	config.BindEnvAndSetDefault("cloud_foundry_bbs.cert_file", "")
 	config.BindEnvAndSetDefault("cloud_foundry_bbs.key_file", "")
+	config.BindEnvAndSetDefault("cloud_foundry_bbs.env_include", []string{})
```

Comment:
We should also document those in the `config_template.yaml` file

Reply:
Done!
Let me know if it needs to be modified.